### DISCUSSION TITLE
What is the purpose of the `:Octo pr/issue search` command?

### DISCUSSION THREAD

**Discussion by calops** [pwntester/octo.nvim#353] [Q&A]
Even after reading the code, I'm a bit dumbfounded at what purpose it serves. Is it any different from doing `:Octo pr list` without any filter?

I would have expected `search` to take additional `query` arguments so that I can easily filter on `author:@me` without having to resort to the global `search` command (which does take those query arguments), which forces me to specify the repository as well.

And while I'm at it, is there currently any way to write a command that means "list all the PRs from the current repository that are assigned to me"?

**Comment by pwntester** [comment]
Hi, the main difference is that `list` will send a request to the server, get a response with all the issues/PRS matching the query and use telescope to pick one. `search` does a live query. Every time you stops typing in the prompt a new request is sent to GitHub and the results are presented in telescope.

`list` filtering options are limited to those available in the GraphQL while `search` ones are the same as running the search in the web UI.

You should be able to call `:lua require("octo.pickers.telescope.provider").picker.search({prompt = "author:@me", static = true})`

**Reply by calops** [reply]
Thanks for this. I still think it should be possible somehow to specify these prompts right from the `:Octo` command.

**Reply by pwntester** [reply]
This should be possible since both `issue` and `pr` versions of search rely on the same API call than the generic one. Will take a look since it may be a bug

**Reply by pwntester** [reply]
This was indeed a bug, let me know if `Octo issue search author:@me` works for you now

**Reply by calops** [reply]
Thanks, it works as expected now
